platform/parsers/elastic_query_dsl/query_translator.go

```diff
@@ -24,7 +26,8 @@ type ClickhouseQueryTranslator struct {
 	Indexes []string
 
 	// TODO this will be removed
-	Table *clickhouse.Table
+	Table     *clickhouse.Table
+	UniqueIDs []string // used for hits queries, to filter out hits that are not in the list of IDs
```

Comment:
Maybe a better name? Filter unique IDs at the end?

Reply:
how about https://github.com/QuesmaOrg/quesma/pull/1435/commits/80dd2667e4ee0de59d99d403942edf471646d836 ?

Reply:
Great!